invenio_theme/assets/semantic-ui/less/invenio_theme/theme/collections/menu.overrides

```diff
@@ -2,17 +2,58 @@
          Invenio Theme Menu Overrides
 ***********************************************/
 
+.ui.inverted.menu {
```

Comment:
Explanation of this change: Moved the order of the styling blocks, so that the backoffice styles are below the main rdm-styling

Reply:
I am not sure I understand what you are trying to achieve by that ? won't it overwrite the entire theme, including the frontsite?

Reply:
The admin-overrides are tied to `.invenio-administration` so it wont affect the frontsite. It was just to organize the file, to avoid confusion for developers. It seems more intuitive to me to have the main styles before the admin ones.